docs/sources/aws_cloudwatch_log_group.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Source: aws_cloudwatch_log_group - Collect logs from AWS CloudWatch Log Groups"
+description: "Allows users to collect logs from AWS CloudWatch Log Groups."
+---
+
+# Source: aws_cloudwatch_log_group - Collect logs from AWS CloudWatch Log Groups
+
+AWS CloudWatch Log Groups are collections of log streams that share the same retention, monitoring, and access control settings. They serve as containers for log data from AWS services, containerized applications, and custom applications.
+
+Using this source, you can collect and analyze logs from CloudWatch Log Groups, enabling real-time monitoring, troubleshooting, and analysis of your AWS resources and applications.
+
+## Example Configurations
+
+### Collect all logs from a Log Group
+
+Collect all logs from a specific CloudWatch Log Group.
+
+```hcl
+connection "aws" "default" {
+  profile = "my-aws-profile"
+}
+
+partition "aws_cloudwatch_log" "application_logs" {
+  source "aws_cloudwatch_log_group" {
+    connection    = connection.aws.default
+    log_group_name = "/aws/lambda/my-function"
```

Comment:
Alignment of `=` looks off, can you please fix it here and in other examples?

Reply:
Done!